app/email_confirmation.html

```diff
@@ -26,10 +26,16 @@
     We will send an email to:
     <br /><span id="success_email"></span>
   </p>
-  Before you can log in, you need to confirm your email address. Next time you
-  check your email, click the verification link to confirm your Account.
+  <p>
+    Before you can log in, you need to confirm your email address. Next time you
+    check your email, click the verification link to confirm your Account.
+  </p>
+  <p>
+    <button id="cancel_button">Cancel email verification</button>
+  </p>
 
-  <script src="js/utils.js" defer></script>
-  <script src="js/email_confirmation_controller.js" defer></script>
+  <script src="js/libs/async_storage.js"></script>
+  <script src="js/user.js"></script>
```

Comment:
We also need sumo_db.
I'm getting a "SumoDB is undefined" error inside `User.is_active` when polling.